microsoft-edge/privacy-whitepaper/index.md

```diff
@@ -374,7 +374,7 @@ The family group organizer may stop the data collection from the family safety p
 <!-- ====================================================================== -->
 ## Find on page 
 
-Use the "Find on page" feature to search a webpage for key words.  In a webpage, press **Ctrl+F** (Windows, Linux) or **Command+F** (macOS).  Or, click **Settings** (**..**), and then select **Find on page**.  A search box opens.  Enter a word, phrase, or other string into this search box.  As you type, all occurrences of the string in the page are highlighted. You can then move through the search results by clicking the **Previous result** and **Next result** buttons.
+Use the "Find on page" feature to search a webpage for key words.  In a webpage, press **Ctrl+F** (Windows, Linux) or **Command+F** (macOS).  Or, click **Settings and more** (**...**), and then select **Find on page**.  A search box opens.  Enter a word, phrase, or other string into this search box.  As you type, all occurrences of the string in the page are highlighted. You can then move through the search results by clicking the **Previous result** and **Next result** buttons.
```

Comment:
this should be: **edge://settings/searchServices**

 **Settings and more** (**...**) > **Settings** > **Search and services**.

Reply:
The **Find on page** command is directly on the **Settings and more** menu, as the existing text above states.
The end of this section has been reworded as gui nav path:
```
To stop Microsoft Edge from finding related matches in **Find on page**, select **Settings and more** (![Settings and more icon](./index-images/settings-and-more-icon.png)) 
> **Settings** > **Search and services**, and then turn off the toggle **Include related matches in Find on page**.
```